sdk/jobs/single-step/pytorch/iris/pytorch-iris.ipynb

```diff
@@ -40,10 +41,10 @@
    "outputs": [],
    "source": [
     "job = CommandJob(\n",
-    "    code=Code(local_path=\"./src\"), #local path where the code is stored\n",
+    "    code_local_path=\"./src\", #local path where the code is stored\n",
     "    command = \"pip install -r requirements.txt && python main.py --iris-csv ${{inputs.iris_csv}} --epochs ${{inputs.epochs}} --lr ${{inputs.lr}}\",\n",
     "    inputs = {\n",
-    "        'iris_csv': 'https://azuremlexamples.blob.core.windows.net/datasets/iris.csv',\n",
+    "        'iris_csv': JobInput(file='https://azuremlexamples.blob.core.windows.net/datasets/iris.csv'),\n",
```

Comment:
are we going to use JobInputs for all non-primitive types?

Reply:
Yes, this simplifies the job inputs/outputs experience for customers. Let me know if you see any issue with this.

Reply:
Couple of questions:
1. How will user differentiate on when to use job inputs vs just use the primitives. For e.g. will an array of strings be a primitive or should it be decorated by a Job Input type?
2. Apart from file, what else can be in a job input constructor?
3. Is Job input the only way for non-primitives? Or can I just do it the old way as well?

Reply:
1. JobInput is for artifact inputs, e.g. dataset and uri. Primitive inputs are passed in as they are. Lists and objects are not supported yet. Once they are, I foresee them being passed in the same way as primitives.
2. See the JobInput spec here. https://github.com/Azure/azureml_run_specification/blob/master/specs/python-sdk/job-python.md
3. Yes, JobInput is the only way.